prod-aws/kafka-shared-msk/billing/transaction-auditor-events.tf

```diff
@@ -0,0 +1,36 @@
+resource "kafka_topic" "transactions_auditor_diff_events" {
+  name               = "billing.transactions-auditor-diff.events"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    "compression.type" = "zstd"
+    # 805 GB
+    "retention.bytes" = "805306368000"
+    # 105 MB
+    "max.message.bytes" = "104857600"
+    # Use tiered storage
+    "remote.storage.enable" = "true"
+    # keep data in hot storage for 2 days
+    "local.retention.ms" = "172800000"
+    # keep data for 60 days
+    "retention.ms"       = "5184000000"
```

Comment:
This needs terraform formatting. 
For handling this automatically see: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/CONTRIBUTING.md